Code.gs

```diff
@@ -230,6 +245,9 @@ function ConvertToCustomEvent(vevent){
   var dtstart = vevent.getFirstPropertyValue('dtstart');
   var dtend = vevent.getFirstPropertyValue('dtend');
   
+  if (dtstart.isDate && dtend.isDate)
```

Comment:
How does that work?

Reply:
Comes from here: http://mozilla-comm.github.io/ical.js/api/ICAL.Time.html (description: `If true, the instance represents a date (as opposed to a date-time)`). Basically, the `DTSTART` (or `DTEND`) object on the `VEVENT` can either be a date or datetime. You can see the discussion about that here: https://github.com/mozilla-comm/ical.js/issues/353

Reply:
Ah fantastic, thanks for digging that up.